README.md

```diff
@@ -212,6 +213,76 @@ See also:
 - [istanbul coverageVariable][istanbul-coverage-variable]
 - [SanboxedModule][sandboxed-module-coverage-variable]
 
+
+### istanbul.enforceThresholds(opt)
+
+Checks whether thresholds pass and fails the gulp task if they fail.
+
+#### opt
+Type: `Object` (optional)
+```js
+{
+  coverageVariable: 'someVariable',
+  thresholds: {
+    global: 60,
+    each: -10
+  }
+}
+```
+
+##### coverageVariable
+Type: `String` (optional)
+Default: `'$$cov_' + new Date().getTime() + '$$'`
+
+The global variable istanbul uses to store coverage
+
+
+##### thresholds
+Type: `Object` (optional)
+Default:
+```
+{
+  global: 100
+}
+```
+
+Thresholds to check the coverage against
+
+Each threshold value can be:
+- A positive number - used as a percentage
+- A negative number - used as the maximum amount of coverage gaps
+- A false value - skips the coverage
```

Comment:
These will work, but it's not the official interface we wish to provide. User shouldn't care about it.